wpilibc/src/main/native/include/frc/controller/PIDController.h

```diff
@@ -233,8 +233,6 @@ class PIDController : public frc::SendableBase {
   void InitSendable(frc::SendableBuilder& builder) override;
 
  protected:
-  mutable wpi::mutex m_thisMutex;
-
   /**
    * Wraps error around for continuous inputs. The original error is returned if
    * continuous mode is disabled. This is an unsynchronized function.
```

Comment:
The comment about it being unsynchronized can be removed since the whole class is now unsynchronized.